README.md

```diff
@@ -73,11 +73,13 @@ The following folks are the maintainers for this repository. Please feel free to
 | Name              | Contact    | Pic                           
 | --------------    | ---------- | ----------------------------- 
 | Samuel Mbuthia    | [@smbuthia](https://github.com/smbuthia) | ![smbuthia][smbuthia]
-| Dusan Milovanovic | [@milovanovicdusan](https://github.com/milovanovicdusan) | ![milovanovicdusan][milovanovicdusan]       
+| Mala Kumar | [@malakumar85](https://github.com/malakumar85) | ![malakumar85][malakumar85] 
+| Catharina Maracke | [@catharinamaracke](https://github.com/catharinamaracke) | ![catharinamaracke][catharinamaracke]
  
 
 [smbuthia]: https://avatars.githubusercontent.com/u/1491710?s=100
-[milovanovicdusan]: https://avatars.githubusercontent.com/milovanovicdusan?s=100
+[malakumar85]: https://avatars.githubusercontent.com/malakumar85?s=100
+[catharinamaracke]: https://avatars.githubusercontent.com/catharinamaracke?s=100
```

Comment:
What if we use html here so that the image size is consistent even when the person's profile picture is not available?
```suggestion
| Samuel Mbuthia    | [@smbuthia](https://github.com/smbuthia) | <img src="https://avatars.githubusercontent.com/smbuthia?s=100" width="100" />
| Mala Kumar | [@malakumar85](https://github.com/malakumar85) | <img src="https://avatars.githubusercontent.com/malakumar85?s=100" width="100" />
| Catharina Maracke | [@catharinamaracke](https://github.com/catharinamaracke) | <img src="https://avatars.githubusercontent.com/catharinamaracke?s=100" width="100" />
```